### DISCUSSION TITLE
Input Dialogue cancel button fix.

### DISCUSSION THREAD

**Discussion by Lil-Kneezy** [TomSchimansky/CustomTkinter#2206] [Q&A]
I was just wondering if there are plans to connect the input dialog box cancel button correctly. As it stands right now clicking any button (including the close window x) passes a successful entry to the next step (i.e. a function that saves whatever was entered into a database. The documentation does not really touch on how to access those buttons or program their funnctions.

**Comment by dipeshSam** [comment]
So, you want to decline the entry if user clicks the native window's close button ('x' type icon on title bar). Is it the case?

I checked! It works well.
It does not save value if user presses `cancel` or `close` button of the window title bar.

It is already done.
Is there any other point you want to mention?
